angr/analyses/reaching_definitions/live_definitions.py

```diff
@@ -17,9 +17,17 @@
 l = logging.getLogger(name=__name__)
 
 
+class GuardUse(Atom):
```

Comment:
this class should be moved into the atoms file since it counts as an atom in the context of def-use